apps/ui/src/ui/Sidebar.tsx

```diff
@@ -1,25 +1,69 @@
 import './Sidebar.css'
 import NavLink from './NavLink'
-import { ReactComponent as Logo } from '../assets/parcelvoy.svg'
-import { Link, NavLinkProps } from 'react-router-dom'
-import { PropsWithChildren, useContext } from 'react'
-import { AdminContext } from '../contexts'
+import { ReactComponent as Logo } from '../assets/logo-icon.svg'
+import { Link, NavLinkProps, useNavigate } from 'react-router-dom'
+import { PropsWithChildren, useCallback, useContext } from 'react'
+import { AdminContext, ProjectContext } from '../contexts'
 import api from '../api'
 import { PreferencesContext } from './PreferencesContext'
+import { Listbox } from '@headlessui/react'
+import { CheckIcon, ChevronUpDownIcon } from './icons'
+import useResolver from '../hooks/useResolver'
+import { LinkButton } from './Button'
 
 interface SidebarProps {
     links?: Array<NavLinkProps & { key: string, icon: string }>
 }
 
 export default function Sidebar({ children, links }: PropsWithChildren<SidebarProps>) {
+    const navigate = useNavigate()
     const profile = useContext(AdminContext)
+    const [project] = useContext(ProjectContext)
     const [preferences, setPreferences] = useContext(PreferencesContext)
+    const [projects] = useResolver(useCallback(async () => await api.projects.all(), []))
```

Comment:
not super important, but for things like this where there are no deps/args, you can always just do:
```tsx
const [projects]
```